client/components/jetpack/sidebar/menu-items/jetpack-cloud.jsx

```diff
@@ -51,7 +51,7 @@ export default ( { path } ) => {
 					} ) }
 					link={ settingsPath( siteSlug ) }
 					onNavigate={ onNavigate }
-					selected={ itemLinkMatches( [ settingsPath( siteSlug ) ], path ) }
+					selected={ itemLinkMatches( settingsPath( siteSlug ), path ) }
```

Comment:
Something seems going wrong here. Clicking `Settings` doesn't work as expected
![CleanShot 2022-03-29 at 20 30 50](https://user-images.githubusercontent.com/12430020/160670967-eb1efec4-c31d-4e60-8fe6-73c3b02523cf.gif)

Reply:
Good catch! Should've been fixed in 907ac96.